completion/exceptions.py

```diff
@@ -0,0 +1,6 @@
+#Python imports
+from __future__ import unicode_literals, absolute_import
+
+class UnavailableCompletionData(Exception):
+    def __init__(self):
+        Exception.__init__(self, "User's enrollment doesn't have completion data.")
```

Comment:
#NIT instead of `User's enrollment ...`, the message is more accurate as `The learner does not have completion data within course {}`. If you can pass in the course_key, it can help logging